Tools/BuildAndPackage.ps1

```diff
@@ -14,18 +26,33 @@ $slnFile = Join-Path $projectRoot "gcloud-powershell.sln"
 $binDir = Join-Path $projectRoot "Google.PowerShell\bin\"
 $debugDir = Join-Path $binDir "Debug"
 
+
 # Folders where the build artifacts are stored, the things to be packaged up.
 $packageDir = Join-Path $binDir "Packaged"
 $powerShellDir = Join-Path $packageDir "PowerShell"
 $gcpsDir = Join-Path $packageDir "GoogleCloudPowerShell"
-$archivePath = Join-Path $packageDir "powershell-0.1.9.zip"
+$archivePath = Join-Path $packageDir "powershell-$normalizedVersion.zip"
 
 # Purge the existing bin directory.
 Write-Host -ForegroundColor Cyan "*** Purging the bindir***"
 if (Test-Path $binDir) {
     Remove-Item $binDir -Recurse -Force
 }
 
+# Change the module manifest version
+$moduleManifestFile = Join-Path $projectRoot "Google.PowerShell\ReleaseFiles\GoogleCloud.psd1"
+$moduleManifestContent = Get-Content $moduleManifestFile -Raw
```

Comment:
I think I would have preferred something that used Import-PowerShellDataFile, but this looks ok.